packages/app/src/apis.ts

```diff
@@ -32,18 +32,29 @@ import {
   configApiRef,
   createApiFactory,
   errorApiRef,
+  fetchApiRef,
   githubAuthApiRef,
+  googleAuthApiRef,
 } from '@backstage/core-plugin-api';
+import {
+  GCalendarApiClient,
+  gcalendarApiRef,
+} from '@backstage/plugin-gcalendar';
 
 export const apis: AnyApiFactory[] = [
   createApiFactory({
     api: scmIntegrationsApiRef,
     deps: { configApi: configApiRef },
     factory: ({ configApi }) => ScmIntegrationsApi.fromConfig(configApi),
   }),
-
   ScmAuth.createDefaultApiFactory(),
 
+  createApiFactory({
+    api: gcalendarApiRef,
+    deps: { authApi: googleAuthApiRef, fetchApi: fetchApiRef },
+    factory: deps => new GCalendarApiClient(deps),
+  }),
```

Comment:
Since `HomePageCalendar` is provided by the plugin, and the plugin exposes this API already, you actually should not need to add it in here. As long as the "static" part of the JSX of the app renders a `HomePageCalendar` widget (no functional components in the way), this api will be instantiated anyway by the framework. Hopefully you will be able to revert this entire file and things work anyway.